test/e2e/fixtures/ngJq/index.html

```diff
@@ -4,6 +4,10 @@
     {{jqueryVersion}}
 
     <script src="../../../../bower_components/jquery/dist/jquery.js"></script>
+    <script type="text/javascript">
+      // Trying to trick `bindJQuery()`
```

Comment:
@Narrezt, I didn't want to be too explicit, since this is basically fixing a bug and it wouldn't be obvious what is does by looking at the corrected code.

Do you think I should add a reference to the issue or the PR (that explains the issue) ?
Or just add a brief description of what the issue was (and how this tests it's fixed) inline ?
<sub>I'm fine with both.</sub>